app/utils/validators/forms/FieldValidator.scala

```diff
@@ -53,6 +53,8 @@ object FieldValidator {
 
   val nonEmpty: String => Boolean = (input: String) => input.trim.nonEmpty
 
+  val isTrue: Boolean => Boolean = (input: Boolean) => input.equals(true)
```

Comment:
Hmm, `input.equals(true)` => if true return true, if false return false, so can you just return `input`? There is no point to check if it's equal to `true`

Reply:
That's true.  Readability?  :-)